src/modules/orders/dtos/notification.dto.ts

```
@@ -0,0 +1,47 @@
+    
+    @IsNotEmpty()
+    status:string
+
+}
+
+export class EmailDto {
+  @IsEmail()
+  @IsNotEmpty()
+  to: string;
+
+  
```

ESLint

[error] Delete `⏎··`

(prettier/prettier)